tools/model_tools/src/openvino/model_zoo/omz_data_downloader.py

```diff
@@ -30,6 +30,12 @@ def copy_data(output_dir):
         str(data_path),
     )
 
+
+def read_dataset_classes(file_name):
+    file_path = _common.PACKAGE_DIR / 'data' / 'dataset_classes' / file_name
```

Comment:
do we need to check that such file exists?

Reply:
check added